modules/theme-tailwind-magento/atoms/ProductLabel.vue

```diff
@@ -1,14 +1,25 @@
 <template>
-  <div class="rounded-lg max-w-max px-2 pb-px bg-gray-500 text-white font-semibold text-xs" :style="label.style">
-    {{ label.text }}
+  <div
+    class="rounded-lg whitespace-nowrap px-2 pb-px bg-gray-500 text-white font-semibold text-xs"
+    :style="label.style"
+  >
+    {{ t(label.text) }}
   </div>
 </template>
 
 <script setup lang="ts">
+import useI18n from '#ioc/composables/useI18n'
 defineProps({
   label: {
     type: Object,
     default: null,
   },
 })
+const { t } = useI18n()
 </script>
+<i18n lang="yaml">
+en-US:
+  Novinka: New item
```

Comment:
This looks like there is some problematic data on BE side. Label text should come already trasnlated from BE. Or if that cant be the case, it should come in english, not czech.